source/_docs/migrate-drupal-gardens.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Migrating a Drupal Gardens Site to Pantheon
+description: Get all the details you need to know to successfully migrate your site away from Drupal Gardens.
+categories: [drupal]
+tags: [migrate]
+keywords: drupal, pantheon, drupal gardens, import
+---
+Drupal Gardens is ending support on August 1, 2016. This article walks you through exporting and migrating your site from Drupal Gardens to Pantheon.
+
+## Export a Site from Drupal Gardens
+
+1. Log in to your [Drupal Gardens account](https://www.drupalgardens.com/user/login).
+2. Go to **My Sites**.
+3. From the Actions menu next to the site you want to export, select **Export site**.
+4. Select "Want to host it on a 3rd party hosting company" as the reason for exporting, and enter an explanation.
+5. Click **Export**. This may take some time depending on the size of your site.
+6. Once it's finished exporting, save the file (.tar.gz archive) to your computer.
+
```

Comment:
Suggest instructing users to first verify the contents of the archive, to ensure all files are present. Their [export documentation](https://www.drupalgardens.com/documentation/site-export) says: 
> Important: Because of file size limits, site archive files for very large Drupal Gardens websites may not include files in the /sites/default/files folder.
If your site archive file has no files in the docroot/sites/default/files folder, complete the following steps to download them from Drupal Gardens:
Ensure that you have installed or have access to the wget utility.
Extract the site archive file to a new location on your local computer.
Open a command window and navigate to the new docroot folder using the cd command.
Run the following command based on your operating system:
Mac OS X
./get_files.sh
Windows
get_files.sh
Drupal Gardens sends your website's remaining files to the docroot/sites/default/files folder.